CRM/Contribute/Form/ContributionPage.php

```diff
@@ -347,7 +347,7 @@ public function endPostProcess() {
       }
 
       CRM_Core_Session::setStatus(ts("'%1' information has been saved.",
-        [1 => CRM_Utils_Array::value('title', CRM_Utils_Array::value($subPage, $this->get('tabHeader')), $className)]
+        [1 => $this->get('tabHeader')[$subPage]['title'] ?? $className]
```

Comment:
I spot checked a couple pages. This seems a bit of a strange fallback but sure why not.